src/api.rs

```diff
@@ -369,16 +373,16 @@ impl fmt::Display for Packet {
 
 impl Context {
   pub fn new_frame(&self) -> Arc<Frame> {
-    Arc::new(Frame::new(self.fi.padded_w, self.fi.padded_h, self.fi.sequence.chroma_sampling))
+    Arc::new(Frame::new(
+      self.config.frame_info.width.align_power_of_two(3),
+      self.config.frame_info.height.align_power_of_two(3),
+      self.config.frame_info.chroma_sampling
+    ))
   }
 
-  pub fn send_frame<F>(&mut self, frame: F) -> Result<(), EncoderStatus>
-  where
-    F: Into<Option<Arc<Frame>>>
-  {
+  pub fn send_frame(&mut self, frame: Option<Arc<Frame>>) -> Result<(), EncoderStatus> {
```

Comment:
Please leave the `Into<Option>` in place.

Reply:
Fixed